src/utils/ADempiere/numberFormat.js

```diff
@@ -0,0 +1,196 @@
+// ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+// Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
+// Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com www.erpya.com
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import { isEmptyValue } from '@/utils/ADempiere/valueUtils.js'
+import {
+  // currencies
+  FIELDS_CURRENCY,
+  //
+  NUMBER, QUANTITY,
+  // integers
+  FIELDS_INTEGER
+} from '@/utils/ADempiere/references.js'
+import store from '@/store'
+
+/**
+ * Get Default currency ISO code
+ */
+export function getCurrency() {
+  return store.getters.getCurrency
+}
+
+export function getStandardPrecision() {
+  return store.getters.getStandardPrecision
+}
+
+/**
+ * Get country code from store
+ */
+export function getCountryCode() {
+  return store.getters.getCountryLanguage
+}
+
+export function formatNumber({
+  value,
+  displayType,
+  currency,
+  country
+}) {
+  if (isEmptyValue(value)) {
+    value = 0
+  }
+
+  let formattedNumber
+  switch (displayType) {
+    case (FIELDS_CURRENCY.includes(displayType) && displayType):
+      formattedNumber = formatPrice(value, currency, country)
+      break
+
+    case NUMBER.id:
+    case QUANTITY.id:
+      formattedNumber = formatQuantity(value)
+      break
+
+    case (FIELDS_INTEGER.includes(displayType) && displayType):
+    default:
+      formattedNumber = formatQuantity(value, true)
+      break
+  }
+
+  return formattedNumber
+}
+
+/**
+ * Get formatted price and show currency
+ * @param {number} value
+ * @param {string} currency default 'USD'
+ * @returns {string} number format with thousands separator, precision, currency format
+ */
+export function formatPrice(value, currency, country) {
+  if (isEmptyValue(value)) {
+    value = 0
+  }
+
+  if (isEmptyValue(currency)) {
+    currency = getCurrency()
+  }
+
+  // const precision = getStandardPrecision()
+
+  if (isEmptyValue(country)) {
+    country = getCountryCode()
+  }
+
+  // get formatted currency number
+  return new Intl.NumberFormat(country, {
```

Comment:
This already exist for valueFormat

Reply:
this was moved to numberFormat to place there all the methods and helpers that have to do with numbers.

Delete https://github.com/adempiere/adempiere-vue/pull/932/files#diff-f8797464a94c1bcbfc855b09cbe72e05ba39484ccbbb0dc3cf42a0db83c89093L165

Add https://github.com/adempiere/adempiere-vue/pull/932/files#diff-f0b105e877af046f431bd18c228f85a91c1215a613551754a261fe981fce9fedR82